packages/runtime/src/next/index.tsx

```diff
@@ -95,7 +95,6 @@ export async function getStaticProps(
 
   const makeswift = new Makeswift(getApiKey(), {
     apiOrigin: getApiOrigin(),
-    unstable_previewData: ctx.previewData,
```

Comment:
Is removing this function the right call? 

This is the deprecated API right?

Reply:
Yeah, we'll want to replace if with the new API, though. We should probably deprecate these functions...

Reply:
I think what you have here is the proper change, though! It means that if you're using these legacy functions you're not using versioning, which I'm good with.